src/components/transactions/TransactionIcon.vue

```diff
@@ -15,40 +11,45 @@ export default {
     transactionGroup: {
       type: String,
       required: true
+    },
+    transactionType: {
+      type: String,
+      required: true
+    }
+  },
+  computed: {
+    icon() {
+      if (this.transactionType.indexOf(` `) !== -1) {
```

Comment:
I still have concerns around these string searches in comparison to some enum. Also `transactionType` is used here as `title` which I think is a bad naming.

Reply:
True! let's do an small refactor (no new features, I promise! ;-)).

Reply:
@faboweb i think it should be in a new PR!